Update README.md (#3)
camaraproject/Template_API_Repository: README.md

<a href="https://github.com/camaraproject/§repo_name§/graphs/contributors" title="Contributors"><img src="https://img.shields.io/github/contributors/camaraproject/§repo_name§?style=plastic"></a>
<a href="https://github.com/camaraproject/§repo_name§" title="Repo Size"><img src="https://img.shields.io/github/repo-size/camaraproject/§repo_name§?style=plastic"></a>
<a href="https://github.com/camaraproject/§repo_name§/blob/main/LICENSE" title="License"><img src="https://img.shields.io/badge/License-Apache%202.0-green.svg?style=plastic"></a>
<a href="https://github.com/camaraproject/§repo_name§/releases/latest" title="Latest Release"><img src="https://img.shields.io/github/release/camaraproject/§repo_name§?style=plastic"></a>

# §repo_name§
Repository to describe, develop, document, and test the §repo_name§ API family

## Scope
* Service APIs for “§repo_name§” (see APIBacklog.md)  
* It provides the customer with the ability to:  
  * §API family description§.
  * NOTE: The scope of this API family should be limited (at least at the first stage) to 4G and 5G.  
* Describe, develop, document, and test the APIs (with 1-2 Telcos)  
* Started: §start date§

> * [Latest Release](https://github.com/camaraproject/§repo_name§/releases/latest)

Reviewer: At the start of a sub project there is no latest release. To be more precise: with the new versioning proposal of Release Management there will be for a longer time only pre-releases. See my proposal for the section "Status and released versions"

Reviewer: PS: the line should here be deleted, it does not belong to the Scope.

Author: In some of the other repo readmes, we moved the "latest release section" to the Contributing section below the mailing list.

Reviewer: The information about the latest release isn't meant in first place for contributors (they contribute to main), but for people from outside, interested in the latest version for their work.

Reviewer: Please consider my proposal for the "Status and released version" section.

> ## Contributing

Reviewer: I agree here with @MarkusKuemmerle and propose the following section (don't know if the comments would work as expected within README.md, but worth a try):

```suggestion
## Status and released versions
<!-- Use/uncomment one or multiple the following options -->
<!-- The sub project has no (pre)releases yet, work in progress is within the main branch -->
<!-- Pre-releases of this sub project are available in https://github.com/camaraproject/§repo_name§/releases -->
<!-- The latest public release is available here: https://github.com/camaraproject/§repo_name§/releases/latest -->
<!-- For changes see [CHANGELOG.md](https://github.com/camaraproject/§repo_name§/blob/main/CHANGELOG.md) -->

## Contributing
```

<a href="https://github.com/camaraproject/§repo_name§/releases/latest" title="Latest Release"><img src="https://img.shields.io/github/release/camaraproject/§repo_name§?style=plastic"></a>

# §repo_name§
Repository to describe, develop, document, and test the §repo_name§ API family

## Scope
* Service APIs for “§repo_name§” (see APIBacklog.md)  
* It provides the customer with the ability to:  
  * §API family description§.
  * NOTE: The scope of this API family should be limited (at least at the first stage) to 4G and 5G.  
* Describe, develop, document, and test the APIs (with 1-2 Telcos)  
* Started: §start date§

## Contributing
* Meetings
    * [Registration](https://wiki.camaraproject.org/x/TQAG) Update this link with your meeting join link
    * Schedule: 
    * Minutes: Access [meeting minutes] (update this link to your wiki page using the "Share" link).
* Mailing List
    * Subscribe / Unsubscribe to the mailing list of this Sub Project <https://lists.camaraproject.org/g/sp-§api_family_abbreviation§>.

>     * A message to all Contributors of this Sub Project can be sent using <sp-§api_family_abbreviation§@lists.camaraproject.org>.

Reviewer: Recipients of the mailing list are not automatically "Contributors" in the sense that they have contributed to the sub project. And it could also hold back people, who are just interested in the API, but don't plan to contribute.

```suggestion
    * A message to the community of this Sub Project can be sent using <sp-§api_family_abbreviation§@lists.camaraproject.org>.
```

* [Latest Release](https://github.com/camaraproject/§repo_name§/releases/latest)

>     * Release Tracking: Follow updates on our [wiki release tracking page] (Link to wiki release tracking page using the "share" link from Confluence)

Reviewer: The release tracking within wiki is meant for release planning and management. The source of truth about existing release should be here in GitHub within CHANGELOG.md and the release notes of GitHub (which are a copy of the CHANGELOG.md content).
